apple/resources.bzl

```diff
@@ -79,3 +83,51 @@ def apple_core_ml_library(name, mlmodel, **kwargs):
         data = [mlmodel],
         **kwargs
     )
+
+def objc_intent_library(
```

Comment:
What required making a language specific version, instead of a general one that we can select the language on?

Reply:
I based it on CoreML rules, and since it required depending on the `Intents` framework, I figure it'd be better to have `objc_` and `swift_` rules. Of course, one can use the rule directly if need be.